### PR TITLE
Add: Match modal

### PR DESCRIPTION
**Pull request by jpirazusta**
Match modal when a user has a match for a recently created target.
Flow:
gif

### CODE REVIEW

File: src/components/MatchModal/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+import Modal from 'components/common/Modal';
+import strings from 'locale';
+import { MATCH_SHAPE } from 'constants/shapes';
+import ovals from 'assets/images/match.png';
+import styles from './styles';
+
+const {
+  MATCH: { modalTitle, message, skip },
+} = strings;
```
Comment (ManuViola77) [inline src/components/MatchModal/index.js:14]: 👏

File: src/components/MatchModal/index.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+const MatchModal = ({ match, onHide }) => (
+  <Modal>
+    <Image source={ovals} />
+    <Text style={styles.title}>{modalTitle}</Text>
+    <Text style={styles.message}>{message}</Text>
+    <View style={styles.matchedUser}>
+      {match.avatar.smallThumbUrl && (
+        <Image source={{ uri: match.avatar.smallThumbUrl }} style={styles.avatar} />
+      )}
```
Comment (ManuViola77) [inline src/components/MatchModal/index.js]: you could use a default profile picture when the user doesn't have one.

**Review by ManuViola77 — APPROVED** [review]
great job! 👏

File: src/components/MatchModal/index.js (hunk 2 above)
Comment (jpirazusta) [reply src/components/MatchModal/index.js]: I agree! Thanks! But there is no such one in the designs...

File: src/constants/shapes.js [hunk 3]
```diff
@@ -17,3 +17,18 @@ export const TARGET_SHAPE = shape({
...
+  firstName: string,
+  lastName: string,
+  fullName: string,
+  username: string,
+  gender: string,
+  avatar: shape({
+    originalUrl: string,
+    normalUrl: string,
+    smallThumbUrl: string,
+  }),
```
Comment (elizabethlofredo) [inline src/constants/shapes.js]: I think at least id, firstname, lastname are required

**Review by elizabethlofredo — APPROVED** [review]

File: src/components/MatchModal/index.js (hunk 2 above)
Comment (ManuViola77) [reply src/components/MatchModal/index.js]: actually there is, in the Edit Profile Screen: 

image

Comment (jpirazusta) [reply src/components/MatchModal/index.js]: Oh, great! Thank you!

**Pull request merged**